regression/cbmc/malloc-may-fail/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE
```

Comment:
:no_entry_sign: Please add `.desc` file _without_ the option, that checks for `VERIFICATION SUCCESSFUL`

Reply:
Added.